<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Store Inventory App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listProducts">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelItemImage">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveProduct">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Selected Product</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Selected product:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProductName">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Quantity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelQuantity">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuNewProduct"/>
    <addaction name="menuEditSelectedProduct"/>
    <addaction name="separator"/>
    <addaction name="menuSaveProducts"/>
    <addaction name="menuLoadProducts"/>
    <addaction name="separator"/>
    <addaction name="menuExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuNewProduct">
   <property name="text">
    <string>New Product</string>
   </property>
  </action>
  <action name="menuEditSelectedProduct">
   <property name="text">
    <string>Edit Selected Product</string>
   </property>
  </action>
  <action name="menuSaveProducts">
   <property name="text">
    <string>Save Products</string>
   </property>
  </action>
  <action name="menuLoadProducts">
   <property name="text">
    <string>Load Products</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
